### PR TITLE
 All the buttons are not in full width during responsive view,  fixed

### PR DESCRIPTION
**Pull request by Rohanxmalik**
issue- In the file tab of patient consultation page, all the buttons are not in full width during responsive view
Required fix -in the mobile responsive view, the buttons in the file tab should be on constant full width

image

image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### <samp>🤖 Generated by Copilot at 93cc2b7</samp>

*  Adjust the width of the `ButtonV2` component to be responsive to screen sizes ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-4b12d1558958a3f190a6b752fcd0dc3412364a61308465dcc60892b2b03bf65cL108-R108))
*  Use the `React.ChangeEvent<HTMLInputElement>` type for the input `onChange` handler in the `FileUpload` component ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1508-R1508))
*  Replace the custom `LinearProgressWithLabel` component with the Material UI `LinearProgress` component in the `FileUpload` component ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1517-R1517))
*  Pass the `deleteAudioBlob` and `handleAudioUpload` functions directly to the `ButtonV2` `onClick` props in the `FileUpload` component, to avoid creating new functions on every render ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1525-R1525), [link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1541-R1539))
*  Add the `w-full` class to the parent div of the buttons in the `FileUpload` component, to align them with the input element ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1581-R1574))
*  Wrap each button in a separate div in the `FileUpload` component, to prevent them from shrinking or expanding based on the available space ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-6624210f1e9cdff909251ba965fafd78918e3d0a95fb569b8f151e8528eca8b6L1600-R1609))
*  Add the `sm:w-full` class to the parent div of the recording UI in the `VoiceRecorder` component, to prevent it from overflowing or getting cut off on small screens ([link](https://github.com/coronasafe/care_fe/pull/5608/files?diff=unified&w=0#diff-8e3baf9867fa0cfaf010498614e08a80e9010c7f23c2542c705b90e0d85f85bcL41-R41))

### CODE REVIEW

File: src/Components/Common/components/ButtonV2.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ const ButtonV2 = ({
 }: ButtonProps) => {
   const className = classNames(
     props.className,
-    "font-medium h-min inline-flex whitespace-pre items-center gap-2 transition-all duration-200 ease-in-out cursor-pointer disabled:cursor-not-allowed disabled:bg-gray-200 disabled:text-gray-500 outline-offset-1",
+    "font-medium h-min w-full md:w-fit inline-flex whitespace-pre items-center gap-2 transition-all duration-200 ease-in-out cursor-pointer disabled:cursor-not-allowed disabled:bg-gray-200 disabled:text-gray-500 outline-offset-1",
```
Comment (rithviknishad) [inline src/Components/Common/components/ButtonV2.tsx]: Adding size classes here will affect all buttons used in the platform. Size classes were not put explicitly to allow flexibility based on where it's used.

Discard this changes and pass `w-full md:w-fit` in classNames instead from where it is required.

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@Rohanxmalik 

1. Which issue is this? Could you use closing tags to link the issue to the PR?
2. Update the title to something meaningful. Have a look at other PRs. It should generally describe what your PR does or introduces.

When a changelog is generated, you can't have "I have fixed the issue. Please review it and merge if it works" in the release notes 😅

File: src/Components/Common/components/ButtonV2.tsx (hunk 1 above)
Comment (Rohanxmalik) [reply src/Components/Common/components/ButtonV2.tsx]: I am really sorry its my first time contributing to open source. Can u please tell me how to put labels, there were only 9 labels available, do I have to create a new label? And you were right I ll make changes. @rithviknishad

Comment (rithviknishad) [reply src/Components/Common/components/ButtonV2.tsx]: Not sure what you meant by "labels". Labels can be managed by members with triage permission only. We'll take care of that 😄 

If you meant closing keywords, have a look at this and other PRs on how it's used. https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

Comment (Rohanxmalik) [reply src/Components/Common/components/ButtonV2.tsx]: I am a bit confused but I have made some changes please review @rithviknishad

**Comment by nihal467** [comment]
@rithviknishad re-review the PR

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Pull request merged**